<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LLANTERIA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(180, 213, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="title">
     <string>DETALLES DE VENTA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>142</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="cliente">
        <property name="styleSheet">
         <string notr="true">color: rgb(46, 248, 160);
border-radius:5px;
border:2px sold green;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>CLIENTE:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(46, 248, 160);
border-radius:5px;
border:2px sold green;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>CANTIDAD:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(46, 248, 160);
border-radius:5px;
border:2px sold green;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>PRECIO:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(46, 248, 160);
border-radius:5px;
border:2px sold green;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>TOTAL:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: rgb(46, 248, 160);
border-radius:5px;
border:2px sold green;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>IGV:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(46, 248, 160);
border-radius:5px;
border:2px sold green;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;

</string>
        </property>
        <property name="text">
         <string>PAGO TOTAL:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lb_pagoTotal">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>131</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(125, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_precio">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>142</y>
       <width>131</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(125, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_total">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>188</y>
       <width>131</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(125, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_igv">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>234</y>
       <width>131</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(125, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_cantidad">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(125, 0,255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_cliente">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>601</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(125, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>1121</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="twgdata">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(86, 0, 21);</string>
       </property>
       <column>
        <property name="text">
         <string>CLIENTE</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CANTIDAD</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PRECIO</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>TOTAL</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>IGV</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PAGO TOTAL</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>281</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="b_nuevo">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
color: rgb(255, 5, 52);
font: 14pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover{
background-color: rgb(170, 255, 127);
}</string>
         </property>
         <property name="text">
          <string>NUEVO</string>
         </property>
         <property name="icon">
          <iconset>
           <activeon>../add.svg</activeon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>29</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bagregar">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
color: rgb(255, 5, 52);
font: 14pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover{
background-color: rgb(170, 255, 127);
}</string>
         </property>
         <property name="text">
          <string>AGREGAR</string>
         </property>
         <property name="icon">
          <iconset>
           <activeon>../imagenes/actualizarest.png</activeon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bsalir">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
color: rgb(255, 5, 52);
font: 14pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover{
background-color: rgb(170, 255, 127);
}</string>
         </property>
         <property name="text">
          <string>SALIR</string>
         </property>
         <property name="icon">
          <iconset>
           <activeon>../imagenes/grabarboleta.png</activeon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
